lib/exceljs.browser.js

```diff
@@ -1,26 +1,9 @@
 /* eslint-disable import/no-extraneous-dependencies,node/no-unpublished-require */
 require('core-js/modules/es.promise');
-require('core-js/modules/es.string.includes');
```

Comment:
@alubbe we should update READMD.md, https://github.com/exceljs/exceljs#es5-imports--.

Reply:
Good catch - added